packages/origin-247-certificate/src/offchain-certificate/config/configuration.ts

```diff
@@ -14,7 +14,12 @@ const getConfiguration = (): Configuration => ({
     REDIS_URL: process.env.REDIS_URL ?? { host: 'localhost', port: 6379 },
     MAX_SYNCHRONIZATION_ATTEMPTS_FOR_EVENT: process.env.MAX_SYNCHRONIZATION_ATTEMPTS_FOR_EVENT
         ? Number(process.env.MAX_SYNCHRONIZATION_ATTEMPTS_FOR_EVENT)
-        : 3
+        : 3,
+    ISSUE_BATCH_SIZE: process.env.ISSUE_BATCH_SIZE ? parseInt(process.env.ISSUE_BATCH_SIZE) : 10,
```

Comment:
```suggestion
    ISSUE_BATCH_SIZE: parseInt(process.env.ISSUE_BATCH_SIZE ?? 10, 10),
```
a little bit simpler maybe and second arg to `parseInt`

Reply:
But if for any reason process.env.ISSUE_BATCH_SIZE returns '', then the result of parseInt will be NaN. Tenary operator prevents that